[2020-07-31 00:14:29,269] {{taskinstance.py:616}} INFO - Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [queued]>
[2020-07-31 00:14:29,291] {{taskinstance.py:616}} INFO - Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [queued]>
[2020-07-31 00:14:29,292] {{taskinstance.py:834}} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 00:14:29,292] {{taskinstance.py:835}} INFO - Starting attempt 2 of 2
[2020-07-31 00:14:29,293] {{taskinstance.py:836}} INFO - 
--------------------------------------------------------------------------------
[2020-07-31 00:14:29,300] {{taskinstance.py:855}} INFO - Executing <Task(MySqlOperator): create_mysql_table> on 2020-07-30T00:00:00+00:00
[2020-07-31 00:14:29,301] {{base_task_runner.py:133}} INFO - Running: ['airflow', 'run', 'store_DAG', 'create_mysql_table', '2020-07-30T00:00:00+00:00', '--job_id', '55', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmp5_hy44i4']
[2020-07-31 00:14:30,050] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table [2020-07-31 00:14:30,047] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=584
[2020-07-31 00:14:30,111] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2020-07-31 00:14:30,112] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   """)
[2020-07-31 00:14:30,440] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table [2020-07-31 00:14:30,437] {{__init__.py:51}} INFO - Using executor LocalExecutor
[2020-07-31 00:14:30,826] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table [2020-07-31 00:14:30,826] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
[2020-07-31 00:14:30,895] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table [2020-07-31 00:14:30,894] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [running]> on host 6097e8989e8b
[2020-07-31 00:14:30,927] {{mysql_operator.py:61}} INFO - Executing: CREATE TABLE IF NOT EXISTS clean_store_transactions(STORE_ID varchar(50), STORE_LOCATION varchar(50), PRODUCT_CATEGORY varchar(50), PRODUCT_ID int, MRP float, CP float, DISCOUNT float, SP float, DATE date);
[2020-07-31 00:14:30,950] {{taskinstance.py:1047}} ERROR - The conn_id `mysql_conn` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
    parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
    conn = self.get_connection(self.mysql_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
[2020-07-31 00:14:30,952] {{taskinstance.py:1076}} INFO - All retries failed; marking task as FAILED
[2020-07-31 00:14:30,972] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table Traceback (most recent call last):
[2020-07-31 00:14:30,973] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/bin/airflow", line 32, in <module>
[2020-07-31 00:14:30,973] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     args.func(args)
[2020-07-31 00:14:30,974] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2020-07-31 00:14:30,974] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     return f(*args, **kwargs)
[2020-07-31 00:14:30,974] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2020-07-31 00:14:30,975] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     _run(args, dag, ti)
[2020-07-31 00:14:30,975] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2020-07-31 00:14:30,975] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     pool=args.pool,
[2020-07-31 00:14:30,975] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-07-31 00:14:30,976] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     return func(*args, **kwargs)
[2020-07-31 00:14:30,976] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
[2020-07-31 00:14:30,976] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     result = task_copy.execute(context=context)
[2020-07-31 00:14:30,976] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
[2020-07-31 00:14:30,977] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     parameters=self.parameters)
[2020-07-31 00:14:30,977] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
[2020-07-31 00:14:30,977] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     with closing(self.get_conn()) as conn:
[2020-07-31 00:14:30,978] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
[2020-07-31 00:14:30,978] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     conn = self.get_connection(self.mysql_conn_id)
[2020-07-31 00:14:30,978] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
[2020-07-31 00:14:30,979] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     conn = random.choice(list(cls.get_connections(conn_id)))
[2020-07-31 00:14:30,979] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
[2020-07-31 00:14:30,979] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     conns = cls._get_connections_from_db(conn_id)
[2020-07-31 00:14:30,979] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-07-31 00:14:30,980] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     return func(*args, **kwargs)
[2020-07-31 00:14:30,980] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
[2020-07-31 00:14:30,980] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table     "The conn_id `{0}` isn't defined".format(conn_id))
[2020-07-31 00:14:30,981] {{base_task_runner.py:115}} INFO - Job 55: Subtask create_mysql_table airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
[2020-07-31 00:14:34,641] {{logging_mixin.py:95}} INFO - [[34m2020-07-31 00:14:34,641[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 1[0m
message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [queued]>
message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [queued]>
message ----> 
--------------------------------------------------------------------------------
message ----> Starting attempt 2 of 2
message ----> 
--------------------------------------------------------------------------------
message ----> Executing <Task(MySqlOperator): create_mysql_table> on 2020-07-30T00:00:00+00:00
message ----> Running: ['airflow', 'run', 'store_DAG', 'create_mysql_table', '2020-07-30T00:00:00+00:00', '--job_id', '59', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmphdvpkz6l']
message ----> Job 59: Subtask create_mysql_table [2020-08-04 21:13:53,370] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=2945
message ----> Job 59: Subtask create_mysql_table /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
message ----> Job 59: Subtask create_mysql_table   """)
message ----> Job 59: Subtask create_mysql_table [2020-08-04 21:13:54,897] {{__init__.py:51}} INFO - Using executor LocalExecutor
message ----> Job 59: Subtask create_mysql_table [2020-08-04 21:13:56,754] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
message ----> Job 59: Subtask create_mysql_table [2020-08-04 21:13:57,012] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.create_mysql_table 2020-07-30T00:00:00+00:00 [running]> on host 0885e5a330eb
message ----> Executing: CREATE TABLE IF NOT EXISTS clean_store_transactions(STORE_ID varchar(50), STORE_LOCATION varchar(50), PRODUCT_CATEGORY varchar(50), PRODUCT_ID int, MRP float, CP float, DISCOUNT float, SP float, DATE date);
message ----> The conn_id `mysql_conn` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
    parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
    conn = self.get_connection(self.mysql_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
message ----> All retries failed; marking task as FAILED
message ----> Job 59: Subtask create_mysql_table Traceback (most recent call last):
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/bin/airflow", line 32, in <module>
message ----> Job 59: Subtask create_mysql_table     args.func(args)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
message ----> Job 59: Subtask create_mysql_table     return f(*args, **kwargs)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
message ----> Job 59: Subtask create_mysql_table     _run(args, dag, ti)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
message ----> Job 59: Subtask create_mysql_table     pool=args.pool,
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
message ----> Job 59: Subtask create_mysql_table     return func(*args, **kwargs)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
message ----> Job 59: Subtask create_mysql_table     result = task_copy.execute(context=context)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
message ----> Job 59: Subtask create_mysql_table     parameters=self.parameters)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
message ----> Job 59: Subtask create_mysql_table     with closing(self.get_conn()) as conn:
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
message ----> Job 59: Subtask create_mysql_table     conn = self.get_connection(self.mysql_conn_id)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
message ----> Job 59: Subtask create_mysql_table     conn = random.choice(list(cls.get_connections(conn_id)))
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
message ----> Job 59: Subtask create_mysql_table     conns = cls._get_connections_from_db(conn_id)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
message ----> Job 59: Subtask create_mysql_table     return func(*args, **kwargs)
message ----> Job 59: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
message ----> Job 59: Subtask create_mysql_table     "The conn_id `{0}` isn't defined".format(conn_id))
message ----> Job 59: Subtask create_mysql_table airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
message ----> [[34m2020-08-04 21:13:59,680[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 1[0m
